### ISSUE TITLE
v3 Compatibility? and project plans? open to help?

### ISSUE THREAD

**Issue by gitinthehole** [ricokahler/sanity-super-pane#76]
Hi!

Wanted to check-in on the status of the project. It looks like work has frozen and the bulk update capability is sorely needed in Sanity. Will the plug-in be migrated to v3?

If the plug-in has been frozen, what is the reason? Is there an alternate plug-in? Other plans from Sanity?

If the plug-in is planning to keep living, are you open to help on this project to keep it moving?

Thanks much in advance!

**Comment by runesandnes** [comment]
Yes, an upgrade of this tool is absolutely necessary for Sanity v3!

**Comment by nwazuo** [comment]
This is one of those super-useful tools that feels like it should be part of the sanity studio already. 
Would definitely love to know what the plans here are and willing to help out too.

**Comment by fluxxus** [comment]
Anything similar to this plugin for V3 maybe? Newcomer to Sanity(coming from Directus/Strapi ) and tbh I am a little confused how something similar isn't part of the core.
